test/equatable_test.dart

```diff
@@ -551,6 +553,14 @@ void main() {
         expect(instanceA != instanceB, true);
         expect(instanceA.hashCode != instanceB.hashCode, true);
       });
+
+      test("should return when contents are same but different kind of List",
```

Comment:
minor but I believe the test description should be:
`should return **true** when contents are the same...`